lib/browser/tag/tag.js

```diff
@@ -171,10 +173,7 @@ function Tag(impl, conf, innerHTML) {
     // parse layout after init. fn may calculate args for nested custom tags
     parseExpressions(dom, self, expressions)
 
-    // unmount automatically when our parent does
-    if (parent) parent.on('unmount', self.unmount)
```

Comment:
@rsbondi this means we need more tests to cover also these cases. Could you please make a pull request adding these tests? I guess it will be trivial to fix these issues having more detailed specs.